README.md

```diff
@@ -129,6 +107,27 @@ calling `markProcessed()`, or the steam must be terminated with an exception. If
 after failing to process a record, and not marking it as processed, then no further records can be checkpointed,
 eventually causing the system to run out of memory.
 
+## Consumer Configuration
+The Kinesis Consumer `ConsumerConfig` can be configured via HOCON configuration, which is common for Akka projects
+```hocon
+consumer {
```

Comment:
`consumer` namespace at the top level may cause collision with other configurations (there may be other libraries doing consumer stuff and configured via HOCON). How about `kinesis-consumer` instead?